code/game/machinery/computer/card.dm

```diff
@@ -478,6 +478,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 						inserted_modify_id.registered_account.account_job = jobdatum // this is a terrible idea and people will grief but sure whatever
 
 					inserted_modify_id.access = ( istype(src, /obj/machinery/computer/card/centcom) ? get_centcom_access(t1) : jobdatum.get_access() )
+					inserted_modify_id.access += inserted_modify_id.sticky_access
```

Comment:
```suggestion
					if(inserted_modify_id.sticky_access)
						inserted_modify_id.access += inserted_modify_id.sticky_access
```
To avoid adding nulls into a list.